filament/backend/include/backend/platforms/WebGPUPlatform.h

```diff
@@ -51,6 +52,13 @@ class WebGPUPlatform final : public Platform {
     // either returns a valid device or panics
     [[nodiscard]] wgpu::Device requestDevice(wgpu::Adapter const& adapter);
 
+    static constexpr size_t MAX_ADAPTER_REQUESTS =
+            /*power preferences*/ 2 * /* backend types */ 4 * /* force fallback */ 2;
```

Comment:
why are you using 4 here?  The number of backends that are supported on each platform could be higher.